public/components/page-dashboard/contribution-month/contribution-month.js

```diff
@@ -3,7 +3,7 @@ import DefineMap from 'can-define/map/';
 import view from './contribution-month.stache';
```

Comment:
# READ THIS!

This is a commit that upgrades bitcentive from a CanJS 4.0 to CanJS 5.0 app.  Similar to the notes in the [CanJS 4.0 upgrade PR](https://github.com/donejs/bitcentive/pull/348/files),  I've added comments to the FIRST instance of a change required to make the application work. Search for justinbmeyer in the page to find all of my comments.

If you something and there's not a comment, look through comments above and you will probably find an explanation. If you don't find one after that, leave a question and I'll explain. Thanks!